src/openrct2/entity/EntityRegistry.cpp

```diff
@@ -10,6 +10,7 @@
 #include "EntityRegistry.h"
 
 #include "../Game.h"
+#include "../GameState.h"
```

Comment:
This include is not needed anymore

Reply:
It will be needed eventually, this can stay I think.

Reply:
I looked through the file and I do not think that the GameState include would ever be used.